docs/zh_CN/minio-client-configuration-files.md

```diff
@@ -67,10 +67,10 @@ cat config.json
 ``hosts``存储将被MinIO Client使用的认证证书。
 
 #### ``config.json.old``
-这个文件保存了以前的配置文件版本细节。
+这个文件保存了以前配置文件的版本细节。
 
 #### ``share``目录
-``share``目录保存MinIO Client ``mc share``命令使用的所有对象的上传和下载URL的元数据信息。
+``share``目录保存MinIO客户端（MinIO client）的``mc share``命令使用到的对象的所有上传和下载URL的元数据信息。
 
 ## 了解更多
 * [MinIO Client完全指南](https://docs.min.io/docs/minio-client-complete-guide)
```

Comment:
This URL points to the English version

Reply:
Thanks for your advice. I'll replace these links to EN versions with those which go to CN versions.